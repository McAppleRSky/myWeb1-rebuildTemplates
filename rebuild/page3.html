<!DOCTYPE html>
<html lang='ru'>
<head>
<link href='css/top-menu-page3.css' rel='stylesheet' type='text/css'>
<link href='css/page3.css' rel='stylesheet' type='text/css'>
<meta content='text/html; charset=utf-8' http-equiv='content-type'>
<link href='favicon.ico' rel='shortcut icon'>
<link href='css/main.css' rel='stylesheet' type='text/css'>
<link href='css/top-menu-1-common.css' rel='stylesheet' type='text/css'>
<link href='css/top-menu-1-hover.css' rel='stylesheet' type='text/css'>
<link href='css/bootstrap.css' rel='stylesheet' type='text/css'>
<link href='css/postmaster.css' rel='stylesheet' type='text/css'>
<style media='screen'></style>
<script charset='utf-8' src='js/gettime.js'></script>
<script charset='utf-8' src='js/cookieHandler.js'></script>
<script charset='utf-8' src='js/pageCounter.js'></script>
<script charset='utf-8' src='js/jquery-1.12.4.js'></script>

<script>
  var htmPage = "page3"; // Учеба
  $(function(){
  	$('.li-menu3').children().attr('href','#');
  	$('.tdMenu10').first().children().attr('href','page2.htm');
  	$('.tdMenu10').last().children().attr('href','page4.htm');
  });
</script>
<title>
Мой персональный Веб-сайт
</title>
</head>
<body>
<header>
<nav id='top-menu-1'>
<script charset='utf-8' src='js/topMenuHandler.js'></script>
<ul>
<li id='li-menu0' onmouseout='ChangeOut(this)' onmouseover='ChangeOver(this)'>
<a href='index.htm'>
<img alt='' class='li-menu-img' src='img/icon/caret.png'>
Главная
<br>
страница
</a>
<ul>
<li class='li-menu1'>
<a href='page1.htm'>
Обо мне
</a>
</li>
<li class='li-menu2'>
<a href='page2.htm'>
Мои
<br>
интересы
</a>
</li>
<li class='li-menu3'>
<a href='page3.htm'>
Учеба
</a>
</li>
<li class='li-menu4'>
<a href='page4.htm'>
Фотоальбом
</a>
</li>
<li class='li-menu5'>
<a href='page5.htm'>
Контакты
</a>
</li>
<li class='li-menu6'>
<a href='page6.htm'>
Тест
<br>
по дисциплине
</a>
</li>
</ul>
</li>
<li class='li-menu1' onmouseout='ChangeOut(this)' onmouseover='ChangeOver(this)'>
<a href='page1.htm'>
<img alt='' class='li-menu-img' src='img/icon/caret.png'>
Обо мне
</a>
</li>
<li class='li-menu2' onmouseout='ChangeOut(this)' onmouseover='ChangeOver(this)'>
<a href='page2.htm'>
<img alt='' class='li-menu-img' src='img/icon/caret.png'>
Мои
<br>
интересы
</a>
<ul>
<li>
<a href='page2.htm#page2-1'>Мое хобби</a>
</li>
<li>
<a href='page2.htm#page2-2'>Мои любимые сайты</a>
</li>
<li>
<a href='page2.htm#page2-3'>Мои любимые книги</a>
</li>
<li>
<a href='page2.htm#page2-4'>Моя любимая музыка</a>
</li>
<li>
<a href='page2.htm#page2-5'>Мои любимые фильмы</a>
</li>
</ul>
</li>
<li class='li-menu3' onmouseout='ChangeOut(this)' onmouseover='ChangeOver(this)'>
<a href='page3.htm'>
<img alt='' class='li-menu-img' src='img/icon/caret.png'>
Учеба
</a>
</li>
<li class='li-menu4' onmouseout='ChangeOut(this)' onmouseover='ChangeOver(this)'>
<a href='page4.htm'>
<img alt='' class='li-menu-img' src='img/icon/caret.png'>
Фотоальбом
</a>
</li>
<li class='li-menu5' onmouseout='ChangeOut(this)' onmouseover='ChangeOver(this)'>
<a href='page5.htm'>
<img alt='' class='li-menu-img' src='img/icon/caret.png'>
Контакты
</a>
</li>
<li class='li-menu6' onmouseout='ChangeOut(this)' onmouseover='ChangeOver(this)'>
<a href='page6.htm'>
<img alt='' class='li-menu-img' src='img/icon/caret.png'>
Тест
<br>
по дисциплине
</a>
</li>
</ul>
</nav>

<figure>
<img alt='фотография Борисова Е.А.' src='img/normal.jpg'>
<figcaption id='idFigcaptionHeader'>
фотография
</figcaption>
</figure>
<h1>
Персональный сайт
</h1>

</header>
<aside>
<h3>здесь могла быть ваша реклама:</h3>
<section>
<h4>рекламодатель1</h4>
<p>
рекламная<br>плоскость 1
</p>
</section>
<section>
<h4>рекламодатель2</h4>
<p>
<q>Телеведущий Владимир Соловьев завел телеграм-канал.</q>
Ссылка для подписания
<a href='t.me/SolovievLive'>https://t.me/SolovievLive</a>
.
</p>
</section>

</aside>
<main>
<article>
<section>
<h3 id='h3Uppercase'>Севстопольский государственный университет</h3>
<h3>Кафедра информационных технологий</h3>
</section>
<section>
<figure>
<figcaption>
<strong>Перечень изучаемых дисциплин</strong>
</figcaption>
<table class='mainTable'>
<caption>Дисциплины по семестрам</caption>
<tr>
<th>№ п/п</th>
<th>Наименование предмета</th>
<th>Тест</th>
</tr>
<tr>
<td class='field-colspan' colspan='2'>1 курс</td>
</tr>
<tr>
<td>1.</td>
<td>Веб-технологии</td>
<td>
<a href='page6.htm'>пройти тест</a>
</td>
</tr>
<tr>
<td>2.</td>
<td>Платформа Java</td>
</tr>
<tr>
<td>3.</td>
<td></td>
</tr>
<tr>
<td>4.</td>
<td></td>
</tr>
<tr>
<td>5.</td>
<td></td>
</tr>
<tr>
<td>6.</td>
<td></td>
</tr>
<tr>
<td>7.</td>
<td></td>
</tr>
<tr>
<td>8.</td>
<td></td>
</tr>
<tr>
<td>9.</td>
<td></td>
</tr>
<tr>
<td>10.</td>
<td></td>
</tr>
<tr>
<td>11.</td>
<td></td>
</tr>
<tr>
<td>12.</td>
<td></td>
</tr>
<tr>
<td>13.</td>
<td></td>
</tr>
<tr>
<td>14.</td>
<td></td>
</tr>
<tr>
<td>15.</td>
<td></td>
</tr>
<tr>
<td>16.</td>
<td></td>
</tr>
</table>
</figure>
</section>
</article>
<section>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
</section>
<section>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
</section>
</main>
<footer>
<table class='fullTableMenu'>
<tr>
<td class='tdMenu10'>
&lt;
<a>
prev
</a>
&gt;
</td>
<td id='tdMenu80'></td>
<td class='tdMenu10'>
&lt;
<a>
next
</a>
&gt;
</td>
</tr>
</table>
®
<time datetime='2019-11-13'>13.11.19 г.</time>
, Борисов Е.А. Все права защищены. © Копирование запрещено
<br>
Автор:
<a href='mailto:mcapple@yandex.ru'>Е.Борисов</a>
<table id='tDateBar'>
<tr>
<td>
<div></div>
</td>
</tr>
<tr>
<td>
<div></div>
</td>
</tr>
</table>

</footer>
</body>
</html>
