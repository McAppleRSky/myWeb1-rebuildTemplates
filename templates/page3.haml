!!!
%html(lang="ru")
	%head
		%link(href="css/top-menu-page3.css" rel="stylesheet" type="text/css")
		%link(href="css/page3.css" rel="stylesheet" type="text/css")
		= Haml::Engine.new(File.read("head.haml")).render
		:javascript
			var htmPage = "page3"; // Учеба
			$(function(){
				$('.li-menu3').children().attr('href','#');
				$('.tdMenu10').first().children().attr('href','page2.htm');
				$('.tdMenu10').last().children().attr('href','page4.htm');
			});
		%script(src="js/pageCounter.js")
		%title
			Мой персональный Веб-сайт
	%body
		%header
			= Haml::Engine.new(File.read("nav.haml")).render
			= Haml::Engine.new(File.read("figure-h1.haml")).render
		%aside
			= Haml::Engine.new(File.read("aside.haml")).render
		%main
			%article
				%section
					%h3#h3Uppercase Севстопольский государственный университет
					%h3 Кафедра информационных технологий
				%section
					%figure
						%figcaption
							%strong Перечень изучаемых дисциплин
						%table.mainTable
							%caption Дисциплины по семестрам
							%tr
								%th № п/п
								%th Наименование предмета
								%th Тест
							%tr
								%td.field-colspan(colspan="2") 1 курс
							%tr
								%td 1.
								%td Веб-технологии
								%td
									%a(href="page6.htm") пройти тест
							%tr
								%td 2.
								%td Платформа Java
							%tr
								%td 3.
								%td
							%tr
								%td 4.
								%td
							%tr
								%td 5.
								%td
							%tr
								%td 6.
								%td
							%tr
								%td 7.
								%td
							%tr
								%td 8.
								%td
							%tr
								%td 9.
								%td
							%tr
								%td 10.
								%td
							%tr
								%td 11.
								%td
							%tr
								%td 12.
								%td
							%tr
								%td 13.
								%td
							%tr
								%td 14.
								%td
							%tr
								%td 15.
								%td
							%tr
								%td 16.
								%td
			%section
				%br
				%br
				%br
				%br
				%br
				%br
				%br
				%br
				%br
				%br
				%br
			%section
				%br
				%br
				%br
				%br
				%br
				%br
				%br
				%br
				%br
				%br
				%br
		%footer
			= Haml::Engine.new(File.read("footer.haml")).render
