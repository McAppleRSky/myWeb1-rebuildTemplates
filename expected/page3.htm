<!doctype html>
<html lang="ru">

<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
	<title>Мой персональный Веб-сайт</title>
	<link rel="shortcut icon" href="favicon.ico" />
	<link rel="stylesheet" href="css/main.css" type="text/css">
	<link rel="stylesheet" href="css/top-menu-1-common.css" type="text/css">
	<link rel="stylesheet" href="css/top-menu-page3.css" type="text/css">
	<link rel="stylesheet" href="css/top-menu-1-hover.css" type="text/css">
	<link rel="stylesheet" href="css/page3.css" type="text/css">
	<link rel="stylesheet" type="text/css" href="css/bootstrap.css">
	<link rel="stylesheet" href="css/postmaster.css">
	<script type="text/javascript">
		var htmPage = "page3"; // Учеба
	</script>
	<script src="js/jquery-1.12.4.js" charset="utf-8"></script>
	<script src="js/gettime.js"></script>
	<script src="js/cookieHandler.js" charset="utf-8"></script>
	<script src="js/pageCounter.js" charset="utf-8"></script>
</head>

<body>
	<header>
		<nav id="top-menu-1">
			<script src="js/topMenuHandler.js"></script>
			<ul>
				<li id="li-menu0" onmouseover="ChangeOver(this)" onmouseout="ChangeOut(this)">
					<a href="index.htm">
						<img class="li-menu-img" src="img/icon/caret.png" alt="" />
						Главная<br>страница</a>
					<ul>
						<li class="li-menu1"><a href="page1.htm">Обо мне</a></li>
						<li class="li-menu2"><a href="page2.htm">Мои<br>интересы</a></li>
						<li class="li-menu3"><a href="#">Учеба</a></li>
						<li class="li-menu4"><a href="page4.htm">Фотоальбом</a></li>
						<li class="li-menu5"><a href="page5.htm">Контакты</a></li>
						<li class="li-menu6"><a href="page6.htm">Тест<br>по дисциплине</a></li>
					</ul>
				</li>
				<li class="li-menu1" onmouseover="ChangeOver(this)" onmouseout="ChangeOut(this)"><a href="page1.htm">
						<img class="li-menu-img" src="img/icon/caret.png" alt="" />
						Обо мне</a></li>
				<li class="li-menu2" onmouseover="ChangeOver(this)" onmouseout="ChangeOut(this)"><a href="page2.htm">
						<img class="li-menu-img" src="img/icon/caret.png" alt="" />
						Мои<br>интересы</a>
					<ul>
						<li><a href="page2.htm#page2-1">Мое хобби</a></li>
						<li><a href="page2.htm#page2-2">Мои любимые сайты</a></li>
						<li><a href="page2.htm#page2-3">Мои любимые книги</a></li>
						<li><a href="page2.htm#page2-4">Моя любимая музыка</a></li>
						<li><a href="page2.htm#page2-5">Мои любимые фильмы</a></li>
					</ul>
				</li>
				<li class="li-menu3" onmouseover="ChangeOver(this)" onmouseout="ChangeOut(this)"><a href="#">
						<img class="li-menu-img" src="img/icon/caret.png" alt="" />
						Учеба</a></li>
				<li class="li-menu4" onmouseover="ChangeOver(this)" onmouseout="ChangeOut(this)"><a href="page4.htm">
						<img class="li-menu-img" src="img/icon/caret.png" alt="" />
						Фотоальбом</a></li>
				<li class="li-menu5" onmouseover="ChangeOver(this)" onmouseout="ChangeOut(this)"><a href="page5.htm">
						<img class="li-menu-img" src="img/icon/caret.png" alt="" />
						Контакты</a></li>
				<li class="li-menu6" onmouseover="ChangeOver(this)" onmouseout="ChangeOut(this)"><a href="page6.htm">
						<img class="li-menu-img" src="img/icon/caret.png" alt="" />
						Тест<br>по дисциплине</a></li>
			</ul>
		</nav>

		<figure>
			<img src="img/normal.jpg" alt="фотография Борисова Е.А." />
			<figcaption id="idFigcaptionHeader">фотография</figcaption>
		</figure>
		<h1>Персональный сайт</h1>
	</header>
	<aside>
		<h3>здесь могла быть ваша реклама:</h3>
		<section>
			<h4>рекламодатель1</h4>
			<p>рекламная<br>плоскость 1</p>
		</section>
		<section>
			<h4>рекламодатель2</h4>
			<p><q>Телеведущий Владимир Соловьев завел телеграм-канал.</q> Ссылка для подписания <a href="t.me/SolovievLive">https://t.me/SolovievLive</a> .</p>
		</section>
	</aside>
	<main>
		<article>
			<section>
				<h3 id="h3Uppercase">Севстопольский государственный университет</h3>
				<h3>Кафедра информационных технологий</h3>
			</section>
			<section>
				<figure>
					<figcaption><strong>Перечень изучаемых дисциплин</strong></figcaption>
					<table class="mainTable">
						<caption>Дисциплины по семестрам</caption>
						<tr>
							<th>№ п/п</th>
							<th>Наименование предмета</th>
							<th>Тест</th>
						</tr>
						<tr>
							<td class="field-colspan" colspan="2">1 курс</td>
						</tr>
						<tr>
							<td>1.</td>
							<td>Веб-технологии</td>
							<td><a href="page6.htm">пройти тест</a></td>
						</tr>
						<tr>
							<td>2.</td>
							<td>Платформа Java</td>
						</tr>
						<tr>
							<td>3.</td>
							<td></td>
						</tr>
						<tr>
							<td>4.</td>
							<td></td>
						</tr>
						<tr>
							<td>5.</td>
							<td></td>
						</tr>
						<tr>
							<td>6.</td>
							<td></td>
						</tr>
						<tr>
							<td>7.</td>
							<td></td>
						</tr>
						<tr>
							<td>8.</td>
							<td></td>
						</tr>
						<tr>
							<td>9.</td>
							<td></td>
						</tr>
						<tr>
							<td>10.</td>
							<td></td>
						</tr>
						<tr>
							<td>11.</td>
							<td></td>
						</tr>
						<tr>
							<td>12.</td>
							<td></td>
						</tr>
						<tr>
							<td>13.</td>
							<td></td>
						</tr>
						<tr>
							<td>14.</td>
							<td></td>
						</tr>
						<tr>
							<td>15.</td>
							<td></td>
						</tr>
						<tr>
							<td>16.</td>
							<td></td>
						</tr>
					</table>
				</figure>
			</section>
		</article>
		<section>
			<br><br><br><br><br><br><br><br><br><br><br>
		</section>
		<section>
			<br><br><br><br><br><br><br><br><br><br><br>
		</section>
	</main>
	<footer>
		<table class="fullTableMenu">
			<tr>
				<td class="tdMenu10">&lt;<a href="page2.htm">prev</a>&gt;</td>
				<td id="tdMenu80"></td>
				<td class="tdMenu10">&lt;<a href="page4.htm">next</a>&gt;</td>
			</tr>
		</table>
		&reg; <time datetime="2019-11-13">13.11.19 г.</time>, Борисов Е.А. Все права защищены. &copy; Копирование запрещено
		<br> Автор: <a href="mailto:mcapple@yandex.ru">Е.Борисов</a>
		<table id="tDateBar">
			<tr>
				<td>
					<div></div>
				</td>
			</tr>
			<tr>
				<td>
					<div></div>
				</td>
			</tr>
		</table>
	</footer>
</body>

</html>
